local.tf

```diff
@@ -54,134 +48,235 @@ locals {
     "ssmmessages",
     "sts"]
   no_proxy = "169.254.169.254,${join(",", formatlist("%s.%s", local.endpoint_services, local.amazon_region_domain))}"
+  emr_subnet_non_capacity_reserved_environments = data.terraform_remote_state.common.outputs.aws_ec2_non_capacity_reservation_region
+
+  emr_capacity_reservation_preference     = local.use_capacity_reservation[local.environment] == true ? "open" : "none"
+  emr_capacity_reservation_usage_strategy = local.use_capacity_reservation[local.environment] == true ? "use-capacity-reservations-first" : ""
+  hive_metastore_backend = {
+    development = "aurora"
+    qa          = "aurora"
+    integration = "aurora"
+    preprod     = "aurora"
+    production  = "aurora"
+  }
 
   ebs_emrfs_em = {
     EncryptionConfiguration = {
       EnableInTransitEncryption = false
-      EnableAtRestEncryption = true
+      EnableAtRestEncryption    = true
       AtRestEncryptionConfiguration = {
 
         S3EncryptionConfiguration = {
-          EncryptionMode = "CSE-Custom"
-          S3Object = "s3://${data.terraform_remote_state.management_artefact.outputs.artefact_bucket.id}/emr-encryption-materials-provider/encryption-materials-provider-all.jar"
+          EncryptionMode             = "CSE-Custom"
+          S3Object                   = "s3://${data.terraform_remote_state.management_artefact.outputs.artefact_bucket.id}/emr-encryption-materials-provider/encryption-materials-provider-all.jar"
           EncryptionKeyProviderClass = "uk.gov.dwp.dataworks.dks.encryptionmaterialsprovider.DKSEncryptionMaterialsProvider"
         }
         LocalDiskEncryptionConfiguration = {
-          EnableEbsEncryption = true
+          EnableEbsEncryption       = true
           EncryptionKeyProviderType = "AwsKms"
-          AwsKmsKey = aws_kms_key.ch_ebs_cmk.arn
+          AwsKmsKey                 = aws_kms_key.ch_ebs_cmk.arn
         }
       }
     }
   }
 
   keep_cluster_alive = {
     development = true
-    qa = false
+    qa          = true
     integration = false
```

Comment:
Can generally leave these false, and override the cluster properties in the CI job